testing/charts/test_bullet_chart.py

```diff
@@ -63,7 +63,7 @@ def chart_data(browser, request):
 
     test_data = TEST_DATA[request.param]
     anchor_id = test_data["anchor"]
-    anchor = browser.element(f".//h3[@id='{anchor_id}']/a")
+    anchor = browser.element(f".//h3[@id='{anchor_id}']")
```

Comment:
This now locates a different element in the DOM?

Reply:
Yes, represent heading instate of anchor. While migration, I found that some anchors are in hidden state with href ([example](https://github.com/RedHatQE/widgetastic.patternfly5/pull/100/files#diff-6908ed748a090d40a3303255549b74439bf41dadaf1806d7798d01d11ef7c296R34)) where playwright failed to apply normal click. 
So for consistency I'm using heading as its just just for highlighting widget in web-ui.